Modules/KusamaMainModule.php

```diff
@@ -17,8 +17,18 @@ function initialize()
         $this->first_block_date = '2019-11-28';
         $this->currency = 'kusama';
         $this->currency_details = ['name' => 'Kusama', 'symbol' => 'KSM', 'decimals' => 12, 'description' => null];
-
+        $this->handles_implemented = true;
+        $this->handles_regex = '^([\w\d ]+)?((//?[^/]+)*)$';
```

Comment:
@Har01d not sure about this, but seems like a good idea:
As substrate chain address can be derived from public key, if you pass the address in polkadot to kusama module it will be decoded and address for Kusama derived.
So any substrate chain address typed in search will result in displaying appropriate address on the substrate chain that 3xpl has.